csrankings-a.csv

```diff
@@ -1572,6 +1572,7 @@ Anhong Guo,University of Michigan,https://guoanhong.com,xSZXqZQAAAAJ
 Ani Calinescu,University of Oxford,http://www.cs.ox.ac.uk/people/ani.calinescu,aCljaCYAAAAJ
 Ani Nenkova,University of Pennsylvania,http://www.cis.upenn.edu/~nenkova,vXQdb5kAAAAJ
 Aniello Cimitile,University of Sannio,https://www.ding.unisannio.it/persone/docenti/cimitile,NOSCHOLARPAGE
+Aniket Bera,Purdue University,https://www.cs.purdue.edu/~ab,q3UdHk4AAAAJ

```

Comment:
Please clarify: is this a 75%+ appointment (for the year) at Purdue?

Reply:
Yes, this is a 75%+ appointment at Purdue.